### ISSUE TITLE
The default fcrepo_syn_default_public_key_path should use the fcrepo_syn_tomcat_home variable

### ISSUE THREAD

**Issue by seth-shaw-unlv** [Islandora-Devops/ansible-role-fcrepo-syn#4]
Currently the default value for fcrepo_syn_default_public_key_path is hardcoded to use Ubuntu's tomcat8. This requires both fcrepo_syn_default_public_key_path and fcrepo_syn_tomcat_home to be updated in group_vars if someone wants to use CentOS.

Instead, fcrepo_syn_default_public_key_path should include the fcrepo_syn_tomcat_home variable which would allow us to set the location once in group_vars. 

Further, since our tomcat8 role can dynamically set the appropriate tomcat8_home value ([once the CentOS PR is merged](https://github.com/Islandora-Devops/ansible-role-tomcat8/pull/3)) we can simply set fcrepo_syn_tomcat_home to use tomcat8_home in group_vars to pass along the correct tomcat8 home without having to update group_vars themselves unless they use another tomcat8 role.

**Comment by whikloj** [comment]
@seth-shaw-unlv I have no problem with this, and I can add it to this existing PR if that is ok?

https://github.com/Islandora-Devops/ansible-role-fcrepo-syn/pull/3

**Comment by seth-shaw-unlv** [comment]
Sounds good to me.

**Comment by whikloj** [comment]
@seth-shaw-unlv I clearly shouldn't have added the fix to this on to #3.

Can you open a PR for this, so it can be merged separate from other issues.

**Comment by seth-shaw-unlv** [comment]
See my [comment on another thread](https://github.com/Islandora-Devops/ansible-role-crayfish/pull/3#issuecomment-379002537).

**Comment by whikloj** [comment]
You are correct that https://github.com/Islandora-Devops/claw-playbook/pull/60 will work, but ideally it would be nice to have the internal variables use the `{{ fcrepo_syn_tomcat_home }}` and then we only need to set `fcrepo_syn_tomcat_home: "{{ tomcat8_home }}"` in claw-playbook. But I'll see if I can dig someone up to merge that PR.
